### ISSUE TITLE
C5 binding exception

### ISSUE THREAD

**Issue by alexeyzimarev** [iftahbe/quartznet-RavenDB#2]
I am getting an exception:

```
=== Pre-bind state information ===
LOG: DisplayName = C5, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
 (Fully-specified)
LOG: Appbase = file:///C:/Dev/Platform/Scheduler/bin/Debug/
LOG: Initial PrivatePath = NULL
Calling assembly : Quartz.Impl.RavenDB, Version=1.0.0.0, Culture=neutral, Public KeyToken=null.
===
LOG: This bind starts in default load context.
LOG: Using application configuration file: C:\Dev\Platform\Scheduler\src\Scheduler\bin\Debug\Scheduler.vshost.exe.Config
LOG: Using host configuration file:
LOG: Using machine configuration file from C:\Windows\Microsoft.NET\Framework\v4.0.30319\config\machine.config.
LOG: Policy not being applied to reference at this time (private, custom, partial, or location-based assembly bind).
LOG: Attempting download of new URL file:///C:/Dev/Platform/Scheduler/bin/Debug/C5.DLL.
WRN: Comparing the assembly name resulted in the mismatch: PUBLIC KEY TOKEN
ERR: Failed to complete setup of assembly (hr = 0x80131040). Probing terminated.
```

It seems you have built the package using non-signed version but when I restore packages, I get signed versions.

**Comment by iftahbe** [comment]
Thanks, will fix.

**Comment by iftahbe** [comment]
I removed the C5 dependency and published a new version 1.0.2 nuget package.
https://www.nuget.org/packages/Quartz.Impl.RavenDB/
